src/main/mainWindow.ts

```diff
@@ -73,6 +73,13 @@ const [addSettingsListener, removeSettingsListeners] = makeSettingsListenerHelpe
 const [addVencordSettingsListener, removeVencordSettingsListeners] = makeSettingsListenerHelpers(VencordSettings);
 
 function initTray(win: BrowserWindow) {
+    const trayClickHandler = Settings.store.clickTrayToShowHide
```

Comment:
if you move this ternary inside the callback, it wont require a restart:
```js
const onTrayClick = () => {
	if (Settings.store.clickTrayToShowHide && win.isVisible())
		win.hide();
	else
		win.show();
}
```

Reply:
Hi! Thanks for reviewing my pr, I have already committed your proposed changes (I don't know how I didn't think of your way of doing it first, it's a much cleaner solution to such a simple feature), sorry for the delay, I don't know how the notification slipped without me noticing.